### DISCUSSION TITLE
Release title

### DISCUSSION THREAD

**Discussion by smarie** [olivierlacan/keep-a-changelog#597] [Ideas]
Hello, and thanks for the great work ! 

If I'm not mistaken, it seems that the format you propose is missing a key part : the release title.
Note how your own releases contain one 😉   : 

![image](https://github.com/olivierlacan/keep-a-changelog/assets/3236794/0d8f59be-5a98-48e6-a548-aaac81fa0376)

Could we consider having an optional title in each changelog section, between the version number and the date ?

meaning that both

## [1.1.1] - 2023-03-05

AND

## [1.1.1] - More betterness (2023-03-05)

would be supported.
Thanks ! (and sorry if this was already reported, my bad)

**Comment by smarie** [comment]
@olivierlacan , any opinion on this ?

**Comment by naught101** [comment]
I agree this would be useful, but I think it would be MUCH better for  consistency if the format was:

```
## [1.1.1] - 2023-03-05 - Changelog Entry Title
```

This way entries without a title can be displayed along side entries with titles and the elements will be side-by-side:

```
## [1.1.2] - 2023-03-06
## [1.1.1] - 2023-03-05 - More betterness
```

---

Tangential issue, I don't really see the need for the `-` before the date, it's already delineated by the brackets. So I personally would prefer:

```
## [1.1.2] 2023-03-06
## [1.1.1] 2023-03-05: More betterness
```

**Reply by ryandesign** [reply]
> I don't really see the need for the `-` before the date, it's already delineated by the brackets

It's not delineated by the brackets. Brackets are Markdown syntax for a hyperlink. The rendered form does not include the brackets, assuming the hyperlink reference has also been correctly added.
